test/unit/util/CustomHeaderNames.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.util
+
+object CustomHeaderNames {
+
+  val X_CLIENT_ID_HEADER_NAME = "X-Client-ID"
+
+}
```

Comment:
perhaps a little ott having its own file - consider maybe a `RequestHeaders` object as part of `TestData`?

Reply:
I'd like to keep it consistent with our other microservices, even if it's only one header.

Reply:
Do some of our services have a `CustomHeaderNames` in test code?